rdr_service/config.py

```diff
@@ -128,6 +128,10 @@
 GENOMIC_API_MODES = ('gem', 'rhp', 'pgx', 'hdr')
 GENOMIC_CVL_SITES = ('co', 'uw', 'bcm')
 GENOMIC_CVL_SITE_BUCKET_MAP = 'cvl_site_bucket_map'
+GENOMIC_GC_SITE_BUCKET_MAP = 'gc_site_bucket_map'
+JH_BUCKET_NAME = "jh_bucket_name"
+BI_BUCKET_NAME = "bi_bucket_name"
```

Comment:
Are we missing some GC buckets here?

Reply:
Would you mind moving them to where we have the other buckets defined?

Reply:
I have these lines with the other buckets now.